### PR TITLE
sJ

### PR DESCRIPTION
**Pull request by MarGabor**
Corrected some things. I'm especially unsure about the vector representations of the embeddings. If you see any flaws, I'll try to correct them by tomorrow.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1 +1,58 @@
-# Title
+# Using OpenAI embeddings for creation of LLM with custom corpus
```
Comment (MarioStanke) [inline docs/README.md:1]:
```suggestion
# Using OpenAI Embeddings in Order to Use an LLM with a Custom Corpus
```

**Comment by MarGabor** [comment]
So, I implemented the changes, but I'm not sure why the overview.png is not updating in the MD file itself. When I click on it, it redirects me to the image, which is displayed correctly. But scrolling through the MD, it's still somehow cached(?) the old PNG. Maybe it will update after some time?

File: docs/overview.png
Comment (MarioStanke) [inline docs/overview.png:1]: crop the image

**Review by MarioStanke — CHANGES_REQUESTED** [review]
Crop the image.

The pdf display problem is on the side of GitHub Markdowns, and it does not work even if the viewers browser supports pdf. Please link to the PDF in the first place.

**Comment by MarGabor** [comment]
Should be alright now. GitHub seems to need quite some time to update the display of the images in MD files. If you like, we can wait for the image to update and I request a new review, once it has.

**Comment by MarGabor** [comment]
Image is now updated.

**Review by MarioStanke — APPROVED** [review]

**Pull request merged**